python/fbprophet/forecaster.py

```diff
@@ -1047,6 +1047,7 @@ def stan_init():
                 params = model.optimizing(
                     dat, init=stan_init, iter=1e4, **kwargs)
             except RuntimeError:
+                kwargs.pop('algorithm')
```

Comment:
```suggestion
                kwargs.pop('algorithm', None)
```
otherwise this will throw a KeyError if algorithm is not in kwargs.